internal/pkg/chrome/chrome.go

```diff
@@ -149,21 +152,23 @@ func isViable(logger xlog.Logger) bool {
 				"Google Chrome headless is not viable as endpoint returned '%v'",
 				err.Error(),
 			)
-			return false
+			return false, err
 		}
 		logger.DebugOpf(
 			op,
 			"Google Chrome headless is viable as endpoint returned '%v'",
 			v,
 		)
-		return true
+		return true, nil
 	}
 	result := false
+	err := error(nil)
```

Comment:
```suggestion
	err := nil
```

Reply:
![image](https://user-images.githubusercontent.com/1389573/93765894-e681cb00-fc15-11ea-9030-11bac4637321.png)

Unfortunately, the short version doesn't work but the long version seems to be ok.

```suggestion
	var err error
```

Reply:
My bad, `var err error` is the right way 👍